dev-docs/modules/wurflRtdProvider.md

```diff
@@ -78,10 +81,11 @@ pbjs.setConfig({
 ### Parameters
 
 {: .table .table-bordered .table-striped }
+
 | Name                      | Type          | Description                                                      | Default               |
 | :------------------------ | :------------ | :--------------------------------------------------------------- |:----------------------|
 | name                      | String        | Real time data module name                                       | Always set to 'wurfl' |
-| waitForIt                 | Boolean       | Set to `true` if an `auctionDelay` is defined (optional)         | `false`               |
+| waitForIt                 | Boolean       | Set to `true` if an `auctionDelay` is defined (required)         | `false`               |
```

Comment:
```suggestion
| waitForIt                 | Boolean       | Set to `true` if an `auctionDelay` is defined (recommended)         | `false`               |
```

Technically not required, but recommend. Right?

Reply:
Unfortunately `auctionDelay` is required because the module performs an external request for WURFL detection. This ensures the detection completes in time before the auction starts.

Reply:
For a second request, the data would be available. Blocking the entire auction for an external call can be a risky thing for a publisher.

The data will still be available on any ad refresh, page navigation or lazy loaded ad. Or even on the first call, if your endpoint is fast enough.

Or am I missing something here?